### PR TITLE
debug sqlite and mongo and mysql based on tests and add testconnection for edge and mysql

### PR DESCRIPTION
**Pull request by alibazregar**

### CODE REVIEW

File: test/command/DbSource/mongo.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        type: "list",
+        run: "atClient",
+      },
+    },
+  ],
+};
+    const context = new RequestContext(setting);
+    context.cancellation = new CancellationToken();
+    const source = new DbSource(il);
+    await source.executeAsync(context);
```
Comment (Qamsari) [inline test/command/DbSource/mongo.js:46]: Please use this type of test
```javascript
const result = await context.waitToGetSourceAsync("test.test");
console.log(result);
```

File: Models/Connection/MySqlConnectionInfo.js [hunk 2]
```diff
@@ -152,7 +163,9 @@ export default class MySqlConnectionInfo extends ConnectionInfo {
    * @throws {Error}
    */
   async executeProcedure(procedureName, parameters) {
-    const query = `CALL ${procedureName}(${Array.isArray(parameters) ? parameters.join(","): ""})`;
```
Comment (Qamsari) [inline Models/Connection/MySqlConnectionInfo.js:155]: Can we use `?` and avoid using string concatenation?

File: test.db
Comment (Qamsari) [inline test.db:1]: It's must be add to project or create in run time?

File: test1.db
Comment (Qamsari) [inline test1.db:1]: It's must be add to project or create in run time?

**Review by Qamsari — COMMENTED** [review]
please review my issue

File: test.db
Comment (alibazregar) [reply test.db:1]: it must be create in runtime ; i can delete and push it

File: test1.db
Comment (alibazregar) [reply test1.db:1]: no this is not necessary; i delete it in the next commit

File: test/command/DbSource/mongo.js (hunk 1 above)
Comment (alibazregar) [reply test/command/DbSource/mongo.js:46]: thanks ; i will add it

File: Models/Connection/MySqlConnectionInfo.js (hunk 2 above)
Comment (alibazregar) [reply Models/Connection/MySqlConnectionInfo.js:155]: fixed; thanks a lot

**Pull request merged**